indico/modules/events/operations.py

```diff
@@ -383,3 +384,14 @@ def sort_reviewing_questions(questions, new_positions):
         field.position = index
     db.session.flush()
     logger.info('Reviewing questions of %r reordered by %r', questions[0].event, session.user)
+
+
+def create_event_request(event, category):
+    assert event.id != category.id
```

Comment:
Did you mean to use `event.category_id` here? (I'd prefer using `event.category != category` though)

Reply:
Yes to both. I mistook the logic here.